tests/integration/integration-test.sh

```diff
@@ -153,6 +165,17 @@ helm_upgrade() {
   echo -e "${SUCCESS}"
 }
 
+helm-repo_upgrade() {
```

Comment:
This doesn't seem to be called anywhere

Reply:
yes, extended here for completeness as otherwise there is no way to use the artifact repo with custom values.yaml and I felt I may just directly increase functionality here. Should I remove?

Reply:
the way the above helm-repo_install works is deploying the artifacthub charts as is. this part allows later upgrade to a specific `values.yaml` to allow testing with non-default settings. I consider this useful, but have not implemented a test yet

Reply:
You consider it useful as in you do use it locally? In that case, I'd see the value but would ask that you explain the use case in a comment

Reply:
see below in line 170. will move 1 line up

Reply:
I think for the shell code, we did comments ahead of their definition.

Would add context to the comment:
When testing locally and you installed Conaisseur via helm-repo_install (why -_ btw?), you can call this function in an adapted or added test (because you cant call this from outside this file, right?) to set ...

Reply:
ok, will remove